packages/devtools_app/test/resolved_uri_manager_test.dart

```diff
@@ -11,143 +11,142 @@ import 'package:mockito/mockito.dart';
 import 'package:vm_service/vm_service.dart';
 
 void main() async {
-  group('ResolvedUriManager', () {
-    const String isolateId = 'anIsolateId';
-    const uri1 = 'this/is/a/uri1';
-    const uri2 = 'this/is/a/uri2';
-    const packageUri1 = 'uri/am/i1';
-    const packageUri2 = 'uri/am/i2';
-    late ResolvedUriManager resolvedUriManager;
-    final service = MockVmService();
+  late ResolvedUriManager resolvedUriManager;
+  final service = MockVmServiceWrapper();
 
+  when(service.getFlagList())
+      .thenAnswer((_) => Future.value(FlagList(flags: [])));
+  when(service.onDebugEvent).thenAnswer((_) {
+    return const Stream.empty();
+  });
+  when(service.onVMEvent).thenAnswer((_) {
+    return const Stream.empty();
+  });
+  when(service.onIsolateEvent).thenAnswer((_) {
+    return const Stream.empty();
+  });
+  when(service.onStdoutEvent).thenAnswer((_) {
+    return const Stream.empty();
+  });
+  when(service.onStderrEvent).thenAnswer((_) {
+    return const Stream.empty();
+  });
```

Comment:
These setup steps have come up in multiple tests. Would it be possible for the MockVmServiceWrapper class to do this when it is initialized?

Reply:
done